### ISSUE TITLE
Sporadic error in Safari

### ISSUE THREAD

**Issue by DanielBickley** [ether/etherpad-lite#3005]
Using Safari, I get these errors only randomly. Has anyone experienced this?

An error occured while loading the pad
NETWORK_ERR: XMLHttpRequest Exception 101: A network error occurred in synchronous requests. in https://www.mysite.com:9001/static/js/require-kernel.js (line 293)

**Comment by DanielBickley** [comment]
Note that this seems to have come up AFTER I changed it so that my containing application did the following.

Before:
(php) session_set_cookie_params(0, '/', 'www.mysite.com', false);

After
(php) session_set_cookie_params(0, '/', 'mysite.com', true);

Notice that I removed the 'www' and also set the https flag to true. I'm reverting to the previous to see if that gets me fixed up. However, I'd sure prefer to use the secure cookie flag if possible. I think there is another bug open on that one already out there.

**Comment by DanielBickley** [comment]
So php session cookie didn't make any change. However, I updated my settings.json so that I could see the DEBUG logging. My Safari console says:

- - - 

WebSocket network error: OSStatus Error -9807: Invalid certificate chain 

[Error] Failed to load resource: An SSL error has occurred and a secure connection to the server cannot be made. (socket.io, line 0)

FORCED TO DISCONNECT

{disconnect: "userdup"}

- - -

So, I'm gonna futz around with the  "ssl" : { "ca" a bit and see what happens.

**Comment by DanielBickley** [comment]
FWIW, I was able to resolve this issue. I had issues entering the "ssl:ca" setting initially and most browsers didn't seem to have an issue, including my initial version of Safari. I'm guessing that I updated Safari and something unique to that installation was crowing about the lack of intermediate cert on SSL. Not having a great understanding of SSL I wasn't able to find much doc help on that front. Most of the SSL documentation in regards to Etherpad that I found were relating to self-signed certs. Anyway, for anybody else out there who may have a similar problem, all I did was take my SSL ca-bundle and crack it open. I separated out the individual intermediate certs there and put them into their own unique files. I then called the two in using the "ssl:ca" array var in the settings JSON. That seemed to do the trick.
